asgiref/sync.py

```diff
@@ -124,6 +119,19 @@ class SyncToAsync:
     def __init__(self, func):
         self.func = func
 
+        if self.executor is None:
+            if "ASGI_THREADS" in os.environ:
+                executor_kwargs = {
+                    "max_workers": int(os.environ["ASGI_THREADS"]),
+                }
+            else:
+                executor_kwargs = {}
+
+            if sys.version_info >= (3, 6):
+                executor_kwargs["thread_name_prefix"] = "sync-to-async-"
```

Comment:
This should use underscores (if desirable after all) - currently results in "sync-to-async-_0".

Reply:
Done.